### ISSUE TITLE
Brave browser on Chromium: 107.0.5304.110 does not render things properly on HTMLCanvasElement

### ISSUE THREAD

**Issue by projektorius96** [brave/brave-browser#26698]
[Brave Software](https://www.linkedin.com/feed/#) (1.45.123 , Chromium: 107.0.5304.110 (Official Build) (64-bit)) compared with [Google](https://www.linkedin.com/feed/#) Chrome (Chromium: 107.0.5304.107 (Official Build) (64-bit)), at the moment of speaking I am experiencing Canvas (2D) rendering issue (working with Konva.js) on the Brave browser, but not on aforementioned version of Google Chrome, or current latest [Microsoft](https://www.linkedin.com/feed/#) edge, [Mozilla](https://www.linkedin.com/feed/#) Firefox respectively

---

Home quality demo [canvas-rendering-issue--user_agents](https://youtu.be/yIbiNOiqFs0)

---
My machine specs in case that helps:
```diff
OS Name	Microsoft Windows 10 Home
Version	10.0.19044 Build 19044
Other OS Description 	Not Available
OS Manufacturer	Microsoft Corporation
System Name	DESKTOP-EJ039FC
System Manufacturer	HP
System Model	HP Pavilion Power Laptop 15-cb0xx
System Type	x64-based PC
System SKU	2WF10EA#ABU
Processor	Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz, 2808 Mhz, 4 Core(s), 8 Logical Processor(s)

```

**Comment by rebron** [comment]
Hi @projektorius96 Can you go into Shields and allow fingerprinting? 

cc: @pilgrim-brave

**Comment by projektorius96** [comment]
@rebron after enabling fingerprinting via Shields rendering got back to normal, rendering works fine, I do see a full grid just like in any other browser (user-agent) . Any chance to explain , or give some hint why disabled fingerprinting could influence such behaviour in layman's terms after all ? Thanks .

**Comment by rebron** [comment]
@projektorius96  Hope this helps a bit: `https://brave.com/privacy-updates/4-fingerprinting-defenses-2.0/#2-fingerprinting-protections-20-farbling-for-great-good`
